docs/tutorial.rst

```diff
@@ -413,7 +413,7 @@ Are you familiar with passing in `*args` and `**kwargs` in Python?::
   >>> args = [1 2]
   >>> kwargs = {"keyword2": 3
   ...           "keyword1": 4}
-  >>> optional_arg(*args, **kwargs)
+  >>> optional-arg(*args, **kwargs)
```

Comment:
All right. Will do.

One more thing, shouldn't it be **Pro tips** and not **Protips** ?
On Jan 15, 2015 1:48 AM, "Berker Peksag" <notifications@github.com> wrote:

> In docs/tutorial.rst
> <https://github.com/hylang/hy/pull/751#discussion-diff-22966650>:
>
> > @@ -413,7 +413,7 @@ Are you familiar with passing in `*args` and `**kwargs` in Python?::
> >    >>> args = [1 2]
> >    >>> kwargs = {"keyword2": 3
> >    ...           "keyword1": 4}
> > -  >>> optional_arg(*args, **kwargs)
> > +  >>> optional-arg(*args, **kwargs)
>
> Yes, please revert it and squash your commits. Otherwise LGTM.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hylang/hy/pull/751/files#r22966650>.
>